factory/wsSovereignNotifierFactory.go

```diff
@@ -41,3 +54,22 @@ func CreatWsSovereignNotifier(cfg config.Config) (process.WSClient, error) {
 
 	return client.CreateWsClient(argsWsClient)
 }
+
+func getDecodedAddresses(addresses []string) ([][]byte, error) {
+	pubKeyConv, err := pubkeyConverter.NewBech32PubkeyConverter(addressLen, core.DefaultAddressPrefix)
```

Comment:
can be received as param on Create

Reply:
`CreateWsSovereignNotifier`is the factory. It creates components, does not receive them